cypress/e2e/create-order.cy.js

```diff
@@ -0,0 +1,63 @@
+const getIframeBody = () => {
+  // get the iframe > document > body
+  // and retry until the body element is not empty
+  return cy.get('#checkout iframe')
+    .its('0.contentDocument.body').should('not.be.empty')
+    .then(cy.wrap)
+}
+
+describe('Create Order Flow', () => {
+  beforeEach(() => {
+    // login user before every test
+    cy.login('foo@bar.com', 'Qwerty123');
+    cy.visit('/');
+    // add 3 products to cart
+    cy.get("[data-cy='add-to-cart-btn']").filter(":lt(3)").each($el => {
+      cy.wrap($el).click();
+    });
+    // go to cart
+    cy.get("[data-cy='cart-btn']").click();
+    // make an order
+    cy.get("[data-cy='order-btn']").click();
+  })
+
+  it('should successfully submits payment form and creates an order', () => {
+    // fill in the form
+    getIframeBody().find('#cardNumber').type("4242 4242 4242 4242");
```

Comment:
**Issue**: Hardcoding credit card numbers in test scripts is not recommended even if they are dummy numbers. It could lead to security best practices being overlooked. <br> **Fix**: Use environment variables or Cypress fixtures to manage test data securely. <br> **Code Suggestion**: 
 ```
 -    getIframeBody().find('#cardNumber').type("4242 4242 4242 4242");
 +    getIframeBody().find('#cardNumber').type(Cypress.env('TEST_CARD_NUMBER'));
 ```